test/common/network/listener_impl_test.cc

```diff
@@ -98,7 +99,7 @@ TEST(ListenerImplTest, NormalRedirect) {
   EXPECT_CALL(listenerDst, newConnection(_, _, _));
   EXPECT_CALL(listener_callbacks2, onNewConnection_(_))
       .WillOnce(Invoke([&](Network::ConnectionPtr& conn) -> void {
-        EXPECT_EQ("127.0.0.1:10001", conn->localAddress().asString());
+        EXPECT_EQ("127.1.2.3:10001", conn->localAddress().asString());
```

Comment:
Since this isn't used (I think) in the test for actual socket stuff, why not pick arbitrary RFC1918 to make it clear?